README.md

```diff
@@ -323,8 +323,8 @@ Specifies the number of total IP addresses that the `ipamd` daemon should attemp
 addresses to keep available at all times, it sets a target number for a floor on how many total IP addresses are allocated. Setting to a
 non-positive value is same as setting this to 0 or not setting the variable.
 
-`MINIMUM_IP_TARGET` is for pre-scaling, `WARM_IP_TARGET` is for dynamic scaling. For example, suppose a cluster has an
-expected pod density of approximately 30 pods per node. If `WARM_IP_TARGET` is set to 30 to ensure there are enough IPs
+`MINIMUM_IP_TARGET` is for pre-scaling, `WARM_IP_TARGET` is for dynamic scaling, and when `MINIMUM_IP_TARGET` is set, it is required that `WARM_IP_TARGET` is also set to positive value. 
```

Comment:
Does this mean we will support WARM_IP_TARGET = 0? Just in case anyone wants to use just the min ip and that will be max number of pods on that node needing IPs from CNI....

Reply:
Yes, we will continue to allow setting `WARM_IP_TARGET=0`, which customers can use in combination with `MINIMUM_IP_TARGET` and max pods to prevent all future ENI allocation